src/templates/core/proposals/view_proposal.html

```diff
@@ -5,161 +5,215 @@
-  <th>Name</th>
-  </tr>
-</thead>
-<tbody>
-{% for editor in proposal.book_editors.all %}
-<tr>
-<td>{{ editor.profile.full_name }}</td>
-</tr>
-{% empty %}
-No book editors have been assigned
-{% endfor %}
-
-</tbody>
-</table>
-<hr>
-<h5> <b>Contract</b>  {% if proposal.contract %}(<a href="{% url 'proposal_contract_manager' proposal.id %}">View</a>) {% endif %}:</h5>
-{% include "core/proposals/contract/contract.html" %}
-<hr>
-
-<div class="col-md-12 other-sub-heading"><h4>External Review Assignments</h4>
-{% if not proposal.status = 'accepted' and not proposal.status = 'declined'%}
-  <a href="{% url 'add_proposal_reviewers' proposal.id %}" class="btn  btn-transparent-white" style="position: absolute;right:20;margin-top:-37px">
-    <i class="fa fa-plus">&nbsp;</i>Add Reviewers</a>
-{% else %}
-{% endif %}
-</div>
-
-{% include "core/proposals/proposal_review_block.html" with review_data=proposal.review_assignments.all %}
-{% include "core/proposals/proposal_editorial_review.html" %}
-
-{% if not proposal.status = 'accepted' or not proposal.status = 'declined'%}
-<hr>
-  <h3>Action</h3>
-  <p>Once you have completed the review process, you can accept, decline or ask the author to submit revisions the proposal. Accepting the proposal will create a submission for the author to complete and allow you to send them some notes on the proposal. Declining will move the proposal to the archive. If revisions are requested, the author will be able to edit and resubmit their proposal.</p>
-  <div class="container col-md-12" style="width:80%;left:10%;right:10%;position:absolute;">
-    
-  {% include "core/proposals/proposal_action_buttons.html" %}
-      </br></br><hr>
-  </div>
-  {% endif %}
+        </h3>
+    </div>
+    <br/><br/>
+    <div class="col-md-12">
+        <br />
+        <table class="table table-bordered">
+            <thead>
+            <tr>
+                <th>Book Title</th>
+                <th>Subtitle</th>
+                <th>Submitting Author/Editor</th>
+                <th>
+                    Principal Contact &nbsp;&nbsp;
+                    <span class="small">
+                    (<a href="{% url 'proposal_assign_view_submitted' proposal.id %}">Change</a>)
+                </span>
+                </th>
+                <th>Status</th>
+                <th>Date Submitted</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>{{ proposal.title }}</td>
+                <td>{{ proposal.subtitle }}</td>
+                <td>{{ proposal.author }}</td>
+                <td>
+                    <a href="{% url 'view_profile_readonly' proposal.owner.pk %}">
+                        {{ proposal.owner.username }}
+                    </a>
+                    <sup>
+                        <a href="core/email.html"
+                           onclick="return popitup(
+                                   '/email/proposal/{{ proposal.id }}/user/{{ proposal.owner.id }}/',
```

Comment:
Can't we use the `{% url blah %}` tag for this?

Reply:
Yeah we probably can but this was just me reformatting existing code so didn't want to open up too much of a can of worms. This same pattern is repeated throughout the codebase.

Reply:
That's a maintenance nightmare waiting to happen, but I won't insist you do it. Someone will have to fix this at some stage, as I, for one, don't like this practice.